### PR TITLE
Adding 3 up to homepage

### LINKED ISSUE #151: Homepage -     custom 3 up variant

### PR DESCRIPTION
**Pull request by gvn**
closes #151

### CODE REVIEW

File: app/react/pages/home/home.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  render() {
+    return (
+      <div id="page-home">
+        <Greeting
+          header="Transforming Science"
+          subHeader="Mozilla Science Lab is a community of researchers, developers, and librarians making research open and accessible." />
+
+        <ThreeUp
+          item1={{text: `Maximizing access to papers, data, code, and materials so anyone can read and contribute`, icon: `/assets/img/icon-access.svg`}}
+          item2={{text: `A community of researchers advocating for openness and collaboration`, icon: `/assets/img/icon-community.svg`}}
```
Comment (alanmoo) [inline app/react/pages/home/home.js]: Is that a gremlin?

File: app/react/pages/home/home.scss [hunk 2]
```diff
@@ -0,0 +1,30 @@
+#page-home {
+  @mixin join-glyph($character) {
```
Comment (alanmoo) [inline app/react/pages/home/home.scss]: A+ use of a mix-in

Comment (gvn) [inline app/react/pages/home/home.scss]: I ❤️ mixins! 👍

**Comment by alanmoo** [comment]
Hey, I wasn't done with that! 😝

**Comment by alanmoo** [comment]
(It's good, but remove the text gremlins)

**Pull request merged**